roles/init_dbserver/tasks/pg_initdb.yml

```diff
@@ -56,4 +59,4 @@
     daemon_reload: true
     state: started
     enabled: true
-  become: true
+  become: true
```

Comment:
Check you editor's EOF setting.  It's producing an unnecessary diff in this instance.